### ISSUE TITLE
wishlist for 2020a merge sprint

### ISSUE THREAD

**Issue by boegel** [easybuilders/easybuild-easyconfigs#10000]
*(keep this list alphabetical please)*

**High priority**
* [ ] ADF 2019.303 (#10190 )
* [ ] BLAST+ 2.10.0
* [x] binutils 2.34 (system: ~~#9998~~ + ~~#10061~~, GCCcore: ~~#10062~~)
* [x] Bison 3.5.3 (system: ~~#10061~~ , GCCcore: ~~#10062~~) (dep for binutils-GCCcore)
* [x] Boost 1.72.0 (gompi: ~~#9981~~, iimpi: ~~#10099~~)
* [x] Boost.Python 1.72.0 ~~#10172~~ (we do still seperate this from Boost, right?)
* [x] cairo 1.16.0 ~~#10143~~
* [x] CMake 3.16.4 (~~#9979~~)
* [ ] CP2K 7.1
* [x] cURL 7.69.1
* [x] DB 18.1.32 ~~#10088~~
* [x] ELPA 2019.11.001 ~~#10142~~
* [x] expat 2.2.9 (~~#10097~~ dep for Perl)
* [x] flex 2.6.4 ~~#10062~~ (dep for binutils-GCCcore)
* [x] fontconfig 2.13.92 (~~#10101~~ dep for X11)
* [x] freetype 2.10.1 (~~#10101~~ dep for fontconfig)
* [x] GCCcore 9.3.0 (~~#10050~~)
* [x] GCC 9.3.0
* [x] GDAL 3.0.4 ~~#10162~~ 
* [ ] gimkl 2020a (#10100)
* [x] GLib 2.64.1 ~~#10143~~
* [x] gperf 3.1 (~~#10101~~ dep for fontconfig)
* [ ] GROMACS 2020.1 (#10026)
* [x] GObject-Introspection 1.64.0 ~~#10143~~
* [x] help2man 1.47.12 ~~#10062~~ (dep for flex-GCCcore)
* [x] HDF5 1.10.6 ~~#10131~~, revert to 1.10.6 ~~#10173~~
* [x] intltool 0.51.0 ~~#10106~~
* [ ] LAMMPS 3Mar2020 (#10191 intel, #10406 foss )
* [x] LLVM 9.0.1 #10133
* [x] M4 1.4.18 ~~#10062~~ (dep for Bison-GCCcore)
* [x] Mesa 20.0.2 (#10175 )
* [x] Meson 0.53.2 ~~#10114~~
* [x] Metis 5.1.0
* [x] MUMPS 5.2.1 ~~#10139~~
* [ ] NAMD (latest)
* [x] ncurses 6.2 (~~#10074~~)
* [x] netCDF 4.7.3 (~~#10135~~)
* [x] netCDF-C++ 4.3.1 (~~#10135~~)
* [x] netCDF-Fortran 4.5.2 (~~#10135~~)
* [x] networkx 2.4 ~~#10119~~
* [x] Ninja 1.10.0 ~~#10114~~
* [ ] NWChem (latest)
* [ ] OpenFOAM 7
* [ ] OpenFOAM v1912
* [x] ParMetis (4.0.3) (gompi, iimpi: ~~#10103~~)
* [x] PCRE 8.44 ~~#10124~~
* [x] Perl 5.30.2 - (~~#10097~~)
* [x] PETSc 3.12.4 (foss: #10178, intel: #10156 )
* [x] pixman 0.38.4 ~~#10143~~
* [x] Plumed 2.6.0 (unless 2.6.1 comes out before we're finished) (*ÅS*) (~~#10187~~)
* [x] Python 2.7.17 (~~#10145~~)
  ~~* or wait for final 2.7.18 release in April 2020? see https://www.python.org/doc/sunset-python-2/~~
* [x] Python 3.8.2 (~~#9949~~)
* [ ] PyTorch 1.4.0 (foss: #10141 )
* [ ] Qt5-5.14.1 (~~#10223~~ )
* [ ] R 3.6.3 (foss: ~~#10200~~)
* [x] SciPy-bundle (Python-3.8.2) (foss: ~~#9999~~ + ~~#10010~~ + ~~#10111~~ + ~~#10118~~, intel: ~~#10112~~ + ~~#10118~~)
* [x] SQLite 3.31.1
* [ ] TensorFlow 2.1.0 (or possibly 2.2)
* [x] util-linux 2.35 (~~#10101~~ dep for fontconfig)
* [ ] WRF 4.1.4
* [ ] WPS 4.1
* [x] X11 20200222 ~~#10113~~ (depends on Meson+Ninja)
* [x] zlib 1.2.11 ~~#10062~~ (dep for binutils-GCCcore)
* [x] GEOS 3.8.1 #10150 (dep for GDAL)
* [x] SWIG 4.0.1 ~~#10147~~ (dep for GEOS)
* [x] JasPer ~~#10152~~ (dep for GDAL)
* [x] PROJ 7.0 ~~#10154~~  (dep for GDAL)
* [x] libgeotiff 1.5.1 ~~#10155~~  (dep for GDAL)

**Lower priority**
* [ ] Armadillo (latest)
* [ ] ATK (latest)
* [ ] CDO (latest)
* [ ] CGAL (latest)
* [ ] CubeLib (latest)
* [x] Eigen (~~#10098~~ 3.3.7 based on GCCcore-9.3.0-based builddeps)
* [x] FFmpeg 4.2.2 ~~#10129~~
* [ ] FLTK (latest)
* [x] FriBidi 1.0.9
* [x] Gdk-Pixbuf 2.40.0 ~~#10148~~
* [x] GSL 2.6
* [ ] GL2PS (latest)
* [x] HTSlib 1.10.2 (#10206)
* [x] Hypre 2.18.2 #10130
* [ ] IPython 7.13.0 (#10208 )
* [x] ImageMagick 7.0.10-1 ~~#10171~~
* [ ] IRkernel (latest)
* [x] LAME 3.100
* [x] libxc 4.3.4 ~~#10149~~ 
* [ ] LittleCMS (latest)
* [x] LibTIFF 4.1.0 ~~#10096~~
* [x] Mako 1.1.2 (~~#10164~~)
* [ ] NCO (latest)
* [ ] OpenEXR (latest)
* [ ] OpenImageIO (latest)
* [x] PyYAML 5.3 ~~#10138~~
* [ ] QuantumESPRESSO (latest)
* [ ] Qwt (latest)
* [x] SCOTCH 6.0.9
* [x] SuiteSparse 5.7.1 ~~#10140~~
* [ ] Trilinos (latest)
* [x] Tcl 8.6.10 (~~#10075~~)
* [x] Tk 8.6.10 ~~#10127~~
* [x] Tkinter 3.8.2 (~~#10146~~)
* [ ] VCFtools (latest)
* [ ] VTK 8.2.0 (#10253)
* [ ] Valgrind 3.15 (#10188 )
* [x] x264 20191217
* [x] x265 3.3
* [x] Yasm 1.3

**Comment by branfosj** [comment]
I am working on:
- PyTorch 1.4.0 TBC
- PyYAML 5.3 #10138
- libyaml 0.2.2 #10138
- Pillow 7.0 #10137

**Comment by akesandgren** [comment]
We should probably use Pillow-SIMD instead of Pillow where ever possible...

**Comment by branfosj** [comment]
> We should probably use Pillow-SIMD instead of Pillow where ever possible...

Yes, so long as we can remove the `-msse4` in the [compile args](https://github.com/uploadcare/pillow-simd/blob/simd/master/setup.py#L734) in places where that'll not work.

**Comment by akesandgren** [comment]
Should be a fairly simple patch to do that i guess...

**Comment by branfosj** [comment]
I've added Python 2.7.17 in #10145

**Comment by bioteamMichael** [comment]
I'm planning to start working on R, though I'm still waiting on the toolchains to build (and with everyone home things are definitely a bit more chaotic......)

**Comment by akesandgren** [comment]
@bioteamMichael @lexming is also working on that, please sync

**Comment by branfosj** [comment]
ImageMagick (7.0.10-1), including LittleCMS (2.9) and GhostScript (9.52) in #10171

**Comment by bioteamMichael** [comment]
Working on dependencies for BLAST+ 2.10.0.

**Comment by bioteamMichael** [comment]
HTSLib in [10206](https://github.com/easybuilders/easybuild-easyconfigs/pull/10206)

**Comment by bioteamMichael** [comment]
VCFtools is in [10207](https://github.com/easybuilders/easybuild-easyconfigs/pull/10207). The last versioned release was 0.1.16, released in 2018. I went with that as the version, but it looks like updates were being checked into "master" on github as of about 10 days ago, so not sure if you want to consider the master branch the latest release, or stay with 0.1.16.

**Comment by akesandgren** [comment]
If there are proper releases we always use them. So 0.1.16 is the correct version to use.

**Comment by branfosj** [comment]
I am working towards Qt5 5.14.1 - various deps:
* snappy (1.1.8) #10219 
* DBus (1.13.12) #10218 
* SCons (3.1.2) #10217 
* double-conversion (3.1.5) #10217 
* libevent (2.1.11) #10220
* PCRE2 (10.34) #10221

At the moment I am having issues building NSS (3.51) - something broke between 3.47 and 3.48.
* NSS (3.47)
* NSPR (4.25)
* Qt5 5.14.1 #10223

**Comment by branfosj** [comment]
* IRkernel (1.1) #10263 
* OpenEXR (2.4.1) #10265 
* OpenImageIO (2.1.12.0) #10265
